app/components/primer/octicon_component.rb

```diff
@@ -16,8 +16,7 @@ class OcticonComponent < Primer::Component
     def initialize(icon:, size: SIZE_DEFAULT, **kwargs)
       @icon, @kwargs = icon, kwargs
 
-      classes = class_names(@kwargs.delete(:class), @kwargs.delete(:classes))
-      @kwargs[:class] = class_names(Primer::Classify.call(**@kwargs)[:class], classes)
+      @kwargs[:class] = class_names(Primer::Classify.call(**@kwargs)[:class], @kwargs.delete(:classes))
```

Comment:
`Classify` should handle `classes` for us, right? I think we can just do:

```suggestion
      @kwargs[:class] = Primer::Classify.call(**@kwargs)[:class]
```

Reply:
@joelhawksley , if we do as suggested, the `classes` property will be passed along to the call to octicon and regress to the problem we are trying to solve.

I simplified the code doing the following:

https://github.com/heynan0/view_components/commit/7bbafe0ad602796d87adf3e16028ef8ca7f13c76